R/map.R

```diff
@@ -65,8 +68,27 @@ map <- function(x, ...) {
 #' @describeIn map plot a 'ppi' object on a map
 #' @export
 map.ppi <- function(x, map, param, alpha = 0.7, xlim, ylim,
-                    zlim = c(-20, 20), ratio, radar.size = 3,
-                    radar.color = "red", n.color = 1000, ...) {
+                    zlim = c(-20, 20), ratio, radar_size = 3, radar.size = 3,
```

Comment:
@stijnvanhoey arguments radar.size and n.color are still present

Reply:
I think this has to do with maintaining backward compatibility.  It is rather confusing though and I wonder whether we shouldn't just break backward compatibility (it's not that we have a very large user base) @peterdesmet what do you think

Reply:
Yeah, @stijnvanhoey told me later that it was for backward compatibility. I would drop it: including these and remembering to eventually remove them will be more confusing for us maintainers, than it being helpful to users. A wrong parameter error in old code is pretty straightforward to debug.

Reply:
see #126